src/cljs/org/broadinstitute/firecloud_ui/page/workspace/submission_details.cljs

```diff
@@ -126,9 +126,50 @@
                    (:workflows props))}])))})
 
 
+(react/defc AbortSubmission
```

Comment:
i think i see what you mean.....one possiblility is to move the "status" (reffering to the  div whose backgroundcolor calls the "color-for-submission") *outside* of the "AbortSubmission" as you mention.